atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/qos/QosCassandraClient.java

```diff
@@ -40,15 +41,19 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.palantir.atlasdb.AtlasDbConstants;
 import com.palantir.atlasdb.keyvalue.api.TableReference;
 import com.palantir.atlasdb.keyvalue.cassandra.CassandraClient;
 import com.palantir.atlasdb.keyvalue.cassandra.CqlQuery;
 import com.palantir.atlasdb.qos.QosClient;
+import com.palantir.atlasdb.transaction.impl.TransactionConstants;
 
 @SuppressWarnings({"all"}) // thrift variable names.
 public class QosCassandraClient implements CassandraClient {
 
     private static final Logger log = LoggerFactory.getLogger(CassandraClient.class);
+    private static final Function<TableReference, Boolean> ZERO_ESTIMATE_DETERMINING_FUNCTION = tRef ->
+            tRef.equals(TransactionConstants.TRANSACTION_TABLE) || tRef.equals(AtlasDbConstants.DEFAULT_METADATA_TABLE);
```

Comment:
Should we exempt persisted locks too? We could try `HiddenTables::isHidden` for the function if you think we should.

Reply:
backup lock goes through this code path and actually releases a not of consumed bytes. I think it shouldn't be rate-limited, but didnt see enough evidence(see it fail due to the limit exeeding) in the tests to exclude it.

```
get_range_slices on table _persisted_locks for keyRange KeyRange(start_key:, end_key:, count:100)
Consuming an estimated bytes of 10000
Now time is 394
Wait time is 419
Recording an adjusted bytes of -9850
```

Also, note that the transaction table is not a hidden table, so HiddenTables::isHidden is not sufficient.

Reply:
Ah, you're right. If we wanted to do this programmatically we should probably check against the `hiddenTables` from AtlasDbConstants actually... (which still doesn't seem to have the default metadata table)

In any case this is fine as it is :)